src/app/components/Embeds/AmpIframeEmbed/index.stories.tsx

```diff
@@ -1,46 +1,51 @@
 import React from 'react';
-import OEmbedLoader from '.';
-import { sampleRiddleProps, sampleFlourishProps } from './fixture';
+import AmpIframeEmbed from '.';
 import { ServiceContextProvider } from '../../../contexts/ServiceContext';
 import { RequestContextProvider } from '../../../contexts/RequestContext';
 import { ARTICLE_PAGE } from '../../../routes/utils/pageTypes';
 import { Services } from '../../../models/types/global';
-import { OEmbedProps } from '../types';
+import { ampParams } from '../types';
 
 const Component = ({
-  props,
-  isAmp,
+  url,
+  parameters,
   service = 'pidgin',
 }: {
-  props: OEmbedProps;
-  isAmp: boolean;
+  url: string;
+  parameters: ampParams;
   service?: Services;
 }) => (
   <RequestContextProvider
     bbcOrigin="https://www.test.bbc.co.uk"
     id="c0000000000o"
-    isAmp={isAmp}
+    isAmp
     isApp={false}
     pageType={ARTICLE_PAGE}
     pathname="/pathname"
     service={service}
     statusCode={200}
   >
     <ServiceContextProvider service={service}>
-      <OEmbedLoader {...props} />
+      <AmpIframeEmbed parameters={parameters} url={url} />
     </ServiceContextProvider>
   </RequestContextProvider>
 );
 
 export default {
-  title: 'Components/Embed Optimo',
-  component: Component,
+  title: 'Components/Embeds/Amp Iframe Embed',
+  component: AmpIframeEmbed,
+  parameters: { chromatic: { disable: true } },
```

Comment:
To stop Riddle ID changing each time we run Chromatic Snapshots (This does mean however that no components in this file will be included in the snapshots)